modal/_utils/mount_utils.py

```diff
@@ -42,13 +39,16 @@ def validate_volumes(
     volumes: Mapping[Union[str, PurePosixPath], Union["_Volume", "_CloudBucketMount"]],
 ) -> Sequence[Tuple[str, Union["_Volume", "_NetworkFileSystem", "_CloudBucketMount"]]]:
     if not isinstance(volumes, dict):
-        raise InvalidError("volumes must be a dict[str, Volume] where the keys are paths")
+        raise InvalidError("volumes must be a dict[str, Volume] where the keys are mount paths")
```

Comment:
Not something you introduced, but is the value type here no longer correct?

Reply:
Yeh it's no longer entirely accurate. Could just be `dict`.